### DISCUSSION TITLE
IP V6

### DISCUSSION THREAD

**Discussion by devtos** [Chia-Network/chia-blockchain#4070] [Q&A]
Hey, is it possible to switch to IPV6? IPV4 and portforwading is blocked by my ISP and IPV4 should be marked as obsolete in modern software. Greetings

**Comment by Akaino** [comment]
IPV4 is far from obsolete in modern software. It might die by 2040 if nothing replaces the protocols by then...

**Reply by devtos** [reply]
how can i downvote this non answer?

**Comment by VictorLaszio** [comment]
If you're using IPv6 you do not need to port forward? You should have enough available IPv6 addresses to provide every device with a fully routable IP address without requiring NAT.

Can you open up ports and protocols through your firewall?
